django/db/backends/oracle/schema.py

```diff
@@ -34,6 +40,30 @@ def quote_value(self, value):
         else:
             return str(value)
 
+    def create_model(self, model):
+        super().create_model(model)
+        if model._meta.db_table_comment:
+            self.execute(self.sql_create_table_comment % {
+                'table': model._meta.db_table,
+                'table_comment': model._meta.db_table_comment.replace('\'', '').replace('\n', ' ')
+            }, [])
```

Comment:
This won't work on Oracle because you're trying to create the same table twice.

Reply:
I guess you may have misunderstood. sql was executed `self.sql_create_table_comment` not `sql_create_table_with_comment`

Reply:
these code was refacted 

~~~Python
 if model._meta.db_table_comment:
       self.execute(
            self.sql_alter_table_comment
            % {
                "table": model._meta.db_table,
                "comment": self.quote_comment_value(model._meta.db_table_comment),
            },
            [],
       )
~~~